<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDota</class>
 <widget class="QDialog" name="ControlDota">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/control_set.png);</string>
  </property>
  <widget class="QLabel" name="wind_label_m">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0033ff;</string>
   </property>
   <property name="text">
    <string>     通风</string>
   </property>
  </widget>
  <widget class="QLabel" name="smg_label_m">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0033ff;</string>
   </property>
   <property name="text">
    <string>      开设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>    开风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>62</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>62</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>62</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/return_ctl.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="wind_off_pushButton_m">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/wind.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="smg_off_pushButton_m">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/smg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="wind_on_pushButton_m">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/wind_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="smg_on_pushButton_m">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/smg_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="alarm_on_pushButton_m">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/alarm_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="alarm_off_pushButton_m">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/alarm.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_label_m">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0033ff;</string>
   </property>
   <property name="text">
    <string>       警报</string>
   </property>
  </widget>
  <widget class="QPushButton" name="light_on_pushButton_m">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/led_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="light_label_m">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0033ff;</string>
   </property>
   <property name="text">
    <string>     补光</string>
   </property>
  </widget>
  <widget class="QPushButton" name="light_off_pushButton_m">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/led.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>71</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/Ae.png);</string>
   </property>
   <property name="title">
    <string>Cortex-A8</string>
   </property>
   <widget class="QPushButton" name="alarm_on_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>55</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/alert_on.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="alarm_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#0033ff;</string>
    </property>
    <property name="text">
     <string> 通知</string>
    </property>
   </widget>
   <widget class="QPushButton" name="light_on_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>40</width>
      <height>54</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>54</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>54</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/bulb_on.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="light_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#0033ff;</string>
    </property>
    <property name="text">
     <string>开灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="light_off_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>40</width>
      <height>54</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>54</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>54</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/bulb_off.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="alarm_off_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>55</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/alert_off.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="steu_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="one_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>一档</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二档</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thr_pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>三档</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
